ll-property-image.html

```diff
@@ -298,63 +320,49 @@
     },
 
     ready: function() {
-      this.imgTitle = this.imgTitle || this.imgName;
-      this.imgDescription = this.imgDescription || '';
+      this.title = this.title || this.name;
+      this.description = this.description || '';
       this.isDefaultImage = this.isDefaultImage || false;
```

Comment:
I was struggling with some of the default values. Part of it might have been a mismatch of properties. I know that I couldn't default the title to the name in the property, but the description seems that it should be able to be defaulted in the property.

Let me play with this and see if I can understand it better.